### DISCUSSION TITLE
Create folders/subfolders with the Define API

### DISCUSSION THREAD

**Discussion by apresmoi** [codesandbox/codesandbox-client#5368] [Q&A] [answered]
Hello,

I was wondering if there is a way to create folders/subfolders with files using the Define API.

PS: CodeSandbox is awesome.

**Comment by SaraVieira** [comment, marked as answer]
Hey!

Yap! You can make it so that the filename is `folder/file` like `src/index.js`

Here is an example:

https://codesandbox.io/s/define-api-get-request-forked-6qlx3?file=/index.js


Also thank you so much :D

**Comment by apresmoi** [comment]
Hello @SaraVieira,

Thank you, this solves my question.

**Reply by SaraVieira** [reply]
Awesome! Also added this info to the docs :)
